src/Bridge/Doctrine/Orm/SubresourceDataProvider.php

```diff
@@ -35,7 +35,7 @@
  *
  * @author Antoine Bluchet <soyuka@gmail.com>
  */
-final class SubresourceDataProvider implements SubresourceDataProviderInterface
+class SubresourceDataProvider implements SubresourceDataProviderInterface
```

Comment:
This is for consistancy with `ItemDataProvider` and `CollectionDataProvider`.

Reply:
It was an error to not add it on them, please revert this change.